dynamic-rapidpro-workspace-settings/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "update-rapidpro-workspace-settings",
+  "version": "1.0.0",
+  "description": "dynamic-rapidpro-workspace Shared GitHub Action",
+  "main": "index.js",
+  "scripts": {
+    "build": "ncc build index.js --license licenses.txt",
+    "test": "mocha test/*.spec.js --timeout 10000 --slow 500"
+  },
+  "author": "Medic Inc.",
```

Comment:
Unsure if we use this in our package.json files

Reply:
Removed.